packages/playwright-core/src/remote/playwrightServer.ts

```diff
@@ -68,12 +69,19 @@ export class PlaywrightServer {
   }
 
   async listen(port: number = 0): Promise<string> {
-    const server = http.createServer((request, response) => {
+    let wsEndpoint: string | undefined;
+    const server = http.createServer((request: http.IncomingMessage, response: http.ServerResponse) => {
+      if (request.method === 'GET' && (request.url === '/json/' || request.url === '/json')) {
```

Comment:
```suggestion
      if (request.method === 'GET' && request.url === '/json') {
```

this is defined by us so we can define a good standard?